man/dracut.8.asc

```diff
@@ -557,6 +557,7 @@ will not be able to boot. Equivalent to "--compress=zstd -15 -q -T0".
 **--regenerate-all**::
     Regenerate all initramfs images at the default location with the kernel
     versions found on the system. Additional parameters are passed through.
+    This is overridden by the **regenerate** configuration option.
```

Comment:
Should also document the new command line option in the man page.

Reply:
There is not any new command line argument, it's just a new config file option.